sdk/scheduler/src/main/java/com/mesosphere/sdk/state/StateStoreCache.java

```diff
@@ -41,13 +41,18 @@
 
     private static StateStoreCache instance = null;
 
+    @VisibleForTesting
     protected final StateStore store;
 
+    @VisibleForTesting
```

Comment:
consistencyCheckForTests() is checking that the cache members remain internally consistent throughout multithreaded tests